### ISSUE TITLE
FE | Profile | Notification Settings | New Page for Mobile Flow

### ISSUE THREAD

**Issue by SKasimow** [department-of-veterans-affairs/va.gov-team#49459]
## Background

Due to the change in flow when setting up a mobile number for Notification preferences, we will be adding a new page to the flow. 

The new URL is : ` /profile/edit/`

## Tasks
- [x] Create new page using the URL ` /profile/edit/`
- [x] Set up new route to house page and generic features
- [x] Set up navigation flow
- [x] Create in-line validation rules
- [x] Link to the following error states: (ticket #49464 under construction)
- [ ] Add in-line validation rules
- [ ] Put behind FF
- [ ] Add GA
- [ ] Conduct Unit tests 
- [ ] Run E2E tests covering current code and regression
- [ ] Notify PM if this needs to be deployed to Staging behind feature flag
  

## Acceptance Criteria
- [ ] Complete all unit testing
- [ ] End-to-end tests showing 0 violations
- [ ] All axe checks pass
- [ ] Notify Product Manager that the change is ready to be validated on Staging
- [ ] Incorporate any changes resulting from Staging review & re-test & have re-validated
- [ ] Confirm PM has opened a new ticket to deploy changes to Production using feature flag, if needed

## Validation
- Who can validate this ticket? (FE, BE, Design, PM)?
- How can this work be validated?
- What updates need to be made (e.g. product outline, use cases, contact center guide)

## Associated Tickets
- #

## How to configure this issue
- Include practice area in title -- e.g., [Design], [FE], [BE]
- Include project/initiative name - e.g., Auth Profile: Address Change Messaging
- Add label for practice area (frontend, backend, design)
- Add label for project if applicable
- Assign to a Profile team member
- Associate with an Epic if applicable
- If creating for yourself: estimate work if enough info exists to do so

**Comment by mtcA6** [comment]
@adamwhitlock1 this looks like it's complete. Should we close this ticket out? 
![image.png](https://images.zenhubusercontent.com/643d7b45c9629eadf8090613/26439bc1-93b2-4b22-aef2-7906facf1476)

**Comment by adamwhitlock1** [comment]
Almost done. We are going to update a few things.

https://github.com/department-of-veterans-affairs/va.gov-team/issues/53547

Once the above ticket is done, then I think we can work on getting a phased rollout going in production.

**Comment by mtcA6** [comment]
closing as a duplicate
